client/src/templates/Challenges/classic/lower-jaw.css

```diff
@@ -0,0 +1,13 @@
+.lower-jaw-icon-bar {
+  overflow: hidden;
+}
+
+.lower-jaw-btn-icon {
+  line-height: 0;
+  margin-right: 2px;
+  float: right;
+}
+
+.lower-jaw-btn-icon:first-child {
+  margin-right: 0;
+}
```

Comment:
They could live in the editor.css similar to other lower jaw styles.

Reply:
done: moved to editor.css and increased margin to 10px